.github/workflows/ci.yml

```diff
@@ -125,35 +137,23 @@ jobs:
           # setup docker containers
           docker-compose up -d --build
 
-      - name: Run cypress
-        run: |
-          cd projects/plio-frontend/
-
-          # delete the node_modules created by the docker
-          echo 'deleting node modules'
-          rm -rf node_modules/
-
-          # install dependencies from the current shell user
-          echo 'running npm'
-          npm install
-          echo 'finished npm install'
-
-          # setup env secrets
-          export cypress_backend_convert_social_auth_token_url=${{ secrets.CYPRESS_PLIO_BACKEND_CONVERT_SOCIAL_AUTH_TOKEN_URL }}
-          export cypress_backend_client_id=${{ secrets.DJANGO_DEFAULT_OAUTH2_CLIENT_ID }}
-          export cypress_backend_client_secret=${{ secrets.DJANGO_DEFAULT_OAUTH2_CLIENT_SECRET }}
-          export cypress_auth_google_refresh_token=${{ secrets.CYPRESS_AUTH_GOOGLE_REFRESH_TOKEN }}
-          export cypress_auth_google_client_id=${{ secrets.GOOGLE_OAUTH2_CLIENT_ID }}
-          export cypress_auth_google_client_secret=${{ secrets.GOOGLE_OAUTH2_CLIENT_SECRET }}
-
-          # run cypress test cases
-          npx cypress run --record --key ${{ secrets.CYPRESS_RECORD_KEY }}
-
-      - name: Coveralls
-        uses: coverallsapp/github-action@master
+      - name: Install TestCafe BrowserStack plugin
+        run: npm install testcafe-browser-provider-browserstack
+
+      - name: Run Windows 10 tests (chrome@96, firefox@95, edge@96)
+        uses: DevExpress/testcafe-action@latest
+        with:
+          args: "'browserstack:chrome@96.0:Windows 10','browserstack:firefox@95.0:Windows 10','browserstack:edge@96.0:Windows 10' projects/plio-frontend/tests/integration --skip-js-errors"
```

Comment:
Yes, I've added a command to see all the browserstack browsers that are available in `End-to-End-Testing.md`. Based on that we can run on Android, iPad etc.

Reply:
In case you want to see, I did a sample run of a test case on BrowserStack Google Pixel 3
https://automate.browserstack.com/dashboard/v2/builds/81e78c0869a0a8cf2b1ec6eb95b1716aad0600d2/sessions/9e668ca4084af89069682f78fe0690f06c0d4986